en/user_guide/media.rst

```diff
@@ -0,0 +1,59 @@
+Media Management
+================
+
+Open Orchestra allows to manage media in the media library panel.
+
+Folder
+------
+
+All files are organised inside folders. It's possible to browse the folder tree to see files by clicking on the left panel.
+
+Create
+~~~~~~
+
+Creating a folder is done by clicking on the "new folder" button in the left panel.
+
+The creation form needs:
+
+* name: name of the folder
+* list of websites: using the folder content
+
+Remove
+~~~~~~
+
+To remove a folder, select the folder and click on the trash button on the right panel.
+
+Manage files
+------------
+
+The add button in the folder's content allows to upload files.
+
+A trash icon appears on media preview hovering to deleting media.
+
+Click on media preview open a page with two panel: crop files and add meta information.
+
+Crop
+~~~~
+
+Image crop offers some options :
+
+* original: keep original format
+* fixed height: adapt height and keep proportion
+* fixed width: adapt width and keep proportion
+* rectangle: reduce to fit in the rectangle format
+
+Once cropping is done the preview is updated and some choices are added:
+
+* manual crop: to adjust your selection manually
+* upload an alternative image: to replace the existing image by another
```

Comment:
another one
i think it should be precised that the replacement is only for the selected thumbnail format.